<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NDSROsettingsDlg</class>
 <widget class="QDialog" name="NDSROsettingsDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>686</width>
    <height>249</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>10</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QDialogButtonBox" name="CMD_BOX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>NDS ROM Directory</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="m_nds_rom_dir">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>27</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>NDSRO Data Directory</string>
    </property>
    <widget class="QLineEdit" name="m_ndsro_data_dir">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>27</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>NDS Memory Card</string>
    </property>
    <widget class="QLineEdit" name="m_nds_memcard_dir">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>27</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CMD_BOX</sender>
   <signal>accepted()</signal>
   <receiver>NDSROsettingsDlg</receiver>
   <slot>CMD_OK_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CMD_BOX</sender>
   <signal>helpRequested()</signal>
   <receiver>NDSROsettingsDlg</receiver>
   <slot>CMD_HELP_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CMD_BOX</sender>
   <signal>rejected()</signal>
   <receiver>NDSROsettingsDlg</receiver>
   <slot>CMD_CANCEL_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CMD_OK_clicked()</slot>
  <slot>CMD_HELP_clicked()</slot>
  <slot>CMD_CANCEL_clicked()</slot>
 </slots>
</ui>
